Resources/Prototypes/_Mono/Entities/SpaceArtillery/Ammo/90mm.yml

```
@@ -0,0 +1,84 @@
+# 90mm Base
+
+- type: entity
+  id: 90mmBulletBase
+  name: shell
+  parent: BaseBulletTrigger
+  abstract: true
+  categories: [ HideSpawnMenu ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)